benchmarks/common.py

```diff
@@ -426,6 +426,12 @@ def maybe_profile(*args, **kwargs):
         ("dev", "name", "batch_size", "speedup"),
         [current_device, current_name, current_batch_size, float(speedup)],
     )
+    headers, data = torchdynamo.utils.compile_times(repr='csv', aggregate=True)
+    output_csv(
+        "compilation_metrics.csv",
```

Comment:
Maybe use the string from output_filename and add "compilation_metrics" at the end? We generate different output files for different backends, dtypes etc. So, we could use the same pattern in the runner.